src/components/Chat/ChatMessage.tsx

```diff
@@ -18,7 +20,7 @@ export function ChatMessage({ message }: Props) {
   return (
     <Row>
       <div className={`${styles.chatMessage} ${styles[roleClass]}`}>
-        <p>{content}</p>
+        <ReactMarkdown children={content} />
```

Comment:
The flex style of the parent div causes the separate HTML elements from the Markdown to line up horizontally instead of vertically.

<img width="1582" alt="Screenshot 2023-05-21 at 07 07 14" src="https://github.com/eras-fyi/ui/assets/6839162/d4e10720-d705-468d-8c89-4175c65ffdad">

<img width="1582" alt="Screenshot 2023-05-21 at 07 07 18" src="https://github.com/eras-fyi/ui/assets/6839162/09ed9b96-b00a-4e90-8271-9e03ba35e131">

Reply:
Finally sorted it out. We can't use `react-markdown@8` because they migrated too fast into experimental stuff, so the `jest` compiler can't work with it (and I don't know how to sort out the transformers so that `jest` will help us, the internet's suggestions don't work).

Now we're using `react-markdown@6` instead and I've sorted out this flex. Yay!